conan_v2.rst

```diff
@@ -71,9 +71,14 @@ available for Conan 2.0.
 Host and build profiles and new cross-building model
 ----------------------------------------------------
 
-Use always build and host profiles. You can enable it by passing ``-pr:b=default`` in the
-command line to most commands. Do not use ``os_build``, ``arch_build`` anywhere in your
-recipes or code.
+Use always build and host profiles.
+You can enable it:
```

Comment:
Maybe we can add extend the sentence a bit? Like: _Conan 1.x will use one profile by default, to start using two profiles, please do the following_ or something like that?

Reply:
sure